app/src/main/res/drawable/all_absences.xml

```diff
@@ -6,5 +6,5 @@
 	android:viewportHeight="24">
 	<path
 		android:fillColor="?android:attr/textColorPrimary"
-		android:pathData="M2,6V8H14V6H2M2,10V12H14V10H2M20.04,10.13C19.9,10.13 19.76,10.19 19.65,10.3L18.65,11.3L20.7,13.35L21.7,12.35C21.92,12.14 21.92,11.79 21.7,11.58L20.42,10.3C20.31,10.19 20.18,10.13 20.04,10.13M18.07,11.88L12,17.94V20H14.06L20.12,13.93L18.07,11.88M2,14V16H10V14H2Z" />
+		android:pathData="M4.034 5.695v2h12v-2h-12m0 4v2h12v-2h-12m17.226 0.944c-0.14 0-0.28 0.06-0.4 0.17l-1 1 2.05 2.05 1-1a0.53 0.53 0 0 0 0-0.77l-1.28-1.28c-0.1-0.1-0.24-0.17-0.38-0.17m-1.97 1.75l-6.07 6.06v2.06h2.06l6.06-6.07-2.05-2.05M4.034 13.695v2h8v-2z" />
```

Comment:
What was actually changed here?

Reply:
![23](https://user-images.githubusercontent.com/36272047/95112139-db17bf00-0740-11eb-8cbe-ce8bf157a9b9.png)

I moved the icon to the right so that it is more in line with the others.

Reply:
Alright, in that case you also have to adjust `all_absences_checked.xml`.